services/online_exam_service/.env

```
@@ -1,18 +1,18 @@
 DB_PORT=5433
 DB_USER=postgres
 DB_PASSWORD=password
-DB_NAME=course_db
+DB_NAME=exam_db
 
 # BLOCKCHAIN CONFIG
 WEB3_PROVIDER_URL=http://127.0.0.1:7545
-GRADE_LEDGER_CONTRACT_ADDRESS=0x06883DaE76786A8eD6d29a570aDEf63606c99CB3
-OWNER_ACCOUNT_PRIVATE_KEY=0x58572e200a5751af31bc884179f18e5defff55455363a39871503e45fe19dcdf
+GRADE_LEDGER_CONTRACT_ADDRESS=0xFf72998a0e9178Aa2890B82F90bAB2De0Cd118b0
+OWNER_ACCOUNT_PRIVATE_KEY=0xc2bff4a3801cd7b46148d6f316e96375f4183abfbea2b014b11364cf0794d836
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OWNER_ACCOUNT_PRIVATE_KEY key should go before the WEB3_PROVIDER_URL key

(UnorderedKey)